spec/imsc-hrm.html

```diff
@@ -223,9 +223,7 @@ <h2>Terms and Definitions</h2>
       data-cite="ttml2#terms-character-information-item">Character Information Item</dfn>.</p>
 
       <p class="note">The term <a>character</a> is for practical purposes
-        the same as a <a>code point</a>.</p>
-
-      <p><dfn data-lt="code point|code points">code point</dfn>. As defined by [[i18n-glossary]].</p>
+        the same as a <dfn>code point</dfn>, as defined by [[i18n-glossary]].</p>
```

Comment:
still normatively referenced here? https://respec.org/docs/#normative-vs-informative-references

Reply:
_if the reference is in a section marked "informative", or an example, note, or figure, then ReSpec automatically makes the reference non-normative._

We can use the explicit `[[?]]` syntax, which is new.

Reply:
Ah, sorry. I've just checked the changed lines only...
Yes, respec will handle reference in note (or any other non-normative part) as informative, and list into informative references section.